w3af/plugins/audit/redos.py

```diff
@@ -59,7 +59,7 @@ def _generate_delay_tests(self, freq, debugging_id):
             for delay_obj in self.get_delays():
                 yield mutant, delay_obj, debugging_id
 
-    def _find_delay_in_mutant(self, (mutant, delay_obj, debugging_id)):
+    def _find_delay_in_mutant(self, mutant, delay_obj, debugging_id):
```

Comment:
And are you sure that the plugin still works as expected? Those () were there for a reason :-)

Reply:
i think that not working. are you sure?